build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id 'org.jetbrains.kotlin.jvm' version '1.3.60'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.61'
```

Comment:
This is the right thing to do of course, but it looks like users might bump in kotlin mismatches once our dependency bleeds into their project -- we've encountered things like this before and just set compatibility boundaries in the README.md.

If a project on kotlin 1.3.50 uses this plugin with a `gradle check`, errors like this occur:
```
Caused by: java.io.InvalidClassException: org.jetbrains.kotlin.incremental.IncrementalModuleInfo; local class incompatible: stream classdesc serialVersionUID = 0, local class serialVersionUID = 7715220919552742947
```